src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -834,250 +834,6 @@ def get_oidc_issuer_profile(self) -> ManagedClusterOIDCIssuerProfile:
 
         return profile
 
-    def _get_enable_azure_keyvault_kms(self, enable_validation: bool = False) -> bool:
-        """Internal function to obtain the value of enable_azure_keyvault_kms.
-
-        This function supports the option of enable_validation. When enabled, if azure_keyvault_kms_key_id is empty,
-        raise a RequiredArgumentMissingError.
-
-        :return: bool
-        """
-        # read the original value passed by the command
-        enable_azure_keyvault_kms = self.raw_param.get("enable_azure_keyvault_kms")
```

Comment:
Should we remove or keep the KMS-related arguments?

Reply:
We must keep them. We could only reuse the decorator and nothing more.